test/cql-pytest/run

```diff
@@ -27,7 +27,16 @@ run.wait_for_services(pid, [
     lambda: run.check_rest_api(ip),
     lambda: check_cql(ip)
 ])
-success = run.run_pytest(sys.path[0], ['--host=' + ip] + sys.argv[1:])
+
+args = ['--host=' + ip]
+if not any(map(lambda x: x.startswith('--replication'), sys.argv)):
+    # The default for --replication is "all". To faciliate iterative
```

Comment:
Note that test/cql-pytest/run was supposed to be a shortcut - you can also run Scylla yourself and then "pytest", but the run script is easier.  After this patch, "run something.py::sometest"  does the good thing (run the test only once) but "pytest something.py::sometest" does what I don't like - run the requested test twice, once with tablets once without.

Reply:
> but "pytest something.py::sometest" does what I don't like - run the requested test twice, once with tablets once without

You can pass a command line param to `pytest` to disable this: `--replication=tablets`. I know you also don't like this.